Automattic/jetpack
# Update the way we generate / verify XMLRPC connections

Fixes #6879 and #6882 and ensures that each verification secret is unique.

#### Testing instructions:
- apply this to one of your testing sites and run through the various connection flows ( see P7rd6c-Me-p2 )
- test that there are no regressions when setting up a publicize connection


#### Proposed changelog entry for your changes: 
Fixes a bug that causes connection errors.
